test/javascripts/integration/components/invite-panel-test.js

```diff
@@ -0,0 +1,22 @@
+import EmberObject, { set } from "@ember/object";
+import componentTest from "helpers/component-test";
+
+moduleForComponent("invite-panel", { integration: true });
+
+componentTest("can_invite_via_email", {
+  template: "{{invite-panel panel=panel}}",
+
+  beforeEach() {
+    set(this.currentUser, "details", { can_invite_via_email: true });
+    const inviteModel = JSON.parse(JSON.stringify(this.currentUser));
```

Comment:
Why is this step required? Are you using `parse(stringify)` to do some sort of deep clone?

Reply:
Yes, that is the use case.